pkg/bosh/converter/bpm_resources.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"strconv"
 
+	"code.cloudfoundry.org/cf-operator/pkg/kube/controllers/statefulset"
```

Comment:
just nitpicking: we group imports by: core, github, k8s, quarks

Reply:
Shouldn't the `lint` not check things like this?  Also, how to ensure this? We have tried to configure `goimport` in that way, but without success.
I think having it part of the `lint` would be quite important to get early feedback and not have to check x files at the end.